docs/aws/README.md

```diff
@@ -132,29 +174,42 @@ spec:
             requests:
               cpu: 1
 EOF
-kubectl scale deployment inflate --replicas 5
-kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l karpenter=controller -o name)
 ```
+## Automatic Node Provisioning 
+
+Scale the previous deployment to 5 replicas, and begin watching the Karpenter
+pod for log events. 
 
-You can see what EC2 instance type was added to your cluster from Karpenter with
 ```bash
-kubectl get no -L "node.kubernetes.io/instance-type"
+kubectl scale deployment inflate --replicas 5
+kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l karpenter=controller -o name)
 ```
 
-If you scale down the deployment replicas the instance will be terminated after 30 seconds (ttlSeconds).
+## Automatic Node Deprovisioning (timeout)
+
+Now, delete the deployment. After 30 seconds (`ttlSecondsAfterEmpty`),
+Karpenter should deprovision the now empty nodes. 
+
 ```bash
-kubectl scale deployment inflate --replicas 0
+kubectl delete deployment inflate
+kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l karpenter=controller -o name)
 ```
 
-Or you can manually delete the node with
+## Manual Node Deprovisioning
+
+If you delete a node with kubectl, Karpenter deprovisions the corresponding instance.
```

Comment:
We need to be a bit careful here. This is actually not supported in kubernetes without karpenter. It's one of our value-adds. This action is actually quite dangerous without Karpenter, since it can leak EC2 resources.

I'd keep the line "Karpenter automatically adds a node finalizer to properly cordon and drain nodes before they are terminated." which specifies why this is safe for folks who are experienced enough to be nervous about this.

Reply:
done